geomesa-features/geomesa-feature-kryo/src/main/scala/org/locationtech/geomesa/features/kryo/impl/KryoFeatureDeserialization.scala

```diff
@@ -48,18 +48,18 @@ trait KryoFeatureDeserialization extends SimpleFeatureSerializer with LazyLoggin
 
 object KryoFeatureDeserialization extends LazyLogging {
 
-  private val inputs  = new SoftThreadLocal[Input]()
-  private val readers = new ThreadLocalCache[String, Array[KryoAttributeReader]](SerializerCacheExpiry)
+  private val inputBytes   = new SoftThreadLocal[Input]()
+  private val inputStreams = new SoftThreadLocal[Input]()
```

Comment:
Ok, I'm stumped.  Why do we have two 'Input's here?

Reply:
one is for getInput for reading an InputStream, the other for getInput for reading a byte array